Branch-SDK/Branch-SDK/BNCPreferenceHelper.h

```diff
@@ -31,8 +31,9 @@
 @property (assign, nonatomic) BOOL explicitlyRequestedReferrable;
 @property (assign, nonatomic) BOOL isReferrable;
 @property (assign, nonatomic) BOOL isDebug;
-@property (assign, nonatomic) BOOL isContinuingUserActivity;
+@property (assign, nonatomic) BOOL shouldWaitForInit;
```

Comment:
Yes. Search where the boolean is used. It's just meant to prevent init from being called before openUrl or continueUserActivity actually receive the URI/Universal Link path.